pom.xml

```diff
@@ -66,6 +66,9 @@
         <netty.version>4.0.40.Final</netty.version>
         <protostuff.version>1.4.4</protostuff.version>
         <protobuf.version>2.4.1</protobuf.version>
+        
+    	<argLine>-Xms1024m -Xmx2048m</argLine>
```

Comment:
this is as replacement to configuration since there is some order to the agents that might override it if placed wrong